tailwind.config.js

```diff
@@ -8,5 +8,6 @@ module.exports = {
 		'./server/**/*.html',
 		'./src/**/*.vue',
 		'./tailwind.purge.safelist.txt',
+		'fake'
```

Comment:
What's this for?

Reply:
It was removed, I added this to re-compile talwind clasess lolz... @mcstover